### ISSUE TITLE
[BUG] ORC incorrect values when reading file in chunks

### ISSUE THREAD

**Issue by ZaidQureshi** [rapidsai/cudf#2633]
**Describe the bug**
I am trying to scan my dataset (stored as an ORC file) looking for the column value for a specific row. The value returned to me is 0 while I know for a fact the column has values for all rows in the dataset. I did a sanity check using pyarrow and pandas and get a valid value for the same row and column. This si the case for all the columns in my dataset that I have tried. Of ntoe is that the first couple of chunks that I read seem to have valid values, but the remaining (majority) of the chunks seem to return 0s. My dataset has 1600420286 rows and I read in chunks of 50000000 rows. My dataset is too big to fit in GPU memory so I have to read in chunks.

**Steps/Code to reproduce bug**
I use the following python script to read the dataset.
```
import os
import numpy as np
import pandas as pd
import cudf
import datetime
import time
from math import ceil, floor

total_rows = 1600420286

n_iter = 50000000
cols =['trip_distance', 'trip_id']

itrs = ceil(total_rows/n_iter)
total_read_io_time = 0
new_row_count = 0

desired_row = 1292783445
for i in range(itrs):
    start = i * n_iter
    n_rows = n_iter if (((i+1) * n_iter) <= total_rows) else n_iter -(((i+1) * n_iter) - total_rows)

    df = cudf.io.orc.read_orc(path, engine='cudf', num_rows=n_rows, skip_rows=start, columns=cols)

    
    if ((i) == (floor(desired_row/n_iter))):
        print(str(i) + " " + str(desired_row%n_iter) + " " + str(floor(desired_row/n_iter)))
print(str(df['trip_distance'][desired_row%n_iter]) + " " + str(df['trip_id'][desired_row%n_iter]));
    
```

The script I use for using pandas/pyarrow for sanity checking is:
```
import os
import numpy as np
import pandas as pd
import datetime
import time
from math import ceil, floor
import pandas as pd
import pyarrow.orc as orc

total_rows = 1600420286
cols =['trip_distance', 'trip_id']
desired_row = 1292783445
with open(path, 'rb') as file:
    data = orc.ORCFile(file)
    df = data.read(cols).to_pandas()
    print(str(df['trip_distance'][desired_row]) + " " + str(df['trip_id'][desired_row])


```

**Expected behavior**
I expect to get the same value for the desired row as given by pandas/pyarrow.

**Environment overview (please complete the following information)**
 - Environment location: Bare-metal
 - Method of cuDF install: from source

**Environment details**
<details><summary>Click here to see environment details</summary><pre>
     
     **git***
     commit 952a5a20edbe8920195beb7effb69af44672ec51 (HEAD -> branch-0.10, origin/branch-0.10, origin/HEAD)
     Merge: 1da2638d5 6f8a2d029
     Author: Thomas Meier <44478121+mjsamoht@users.noreply.github.com>
     Date:   Fri Aug 16 14:41:10 2019 -0700
     
     Merge pull request #2618 from j-ieong/bug-ext-io-readers-mmapleak
     
     [REVIEW] IO Readers: Fix datasource memory map failure for multiple reads
     **git submodules***
     b165e1fb11eeea64ccf95053e40f2424312599cc thirdparty/cub (v1.7.1)
     97e046f4f4883dc5612dd14bc063e53dd623a6cf thirdparty/jitify (remotes/origin/feature/api_v2)
     
     ***OS Information***
     DISTRIB_ID=Ubuntu
     DISTRIB_RELEASE=18.04
     DISTRIB_CODENAME=bionic
     DISTRIB_DESCRIPTION="Ubuntu 18.04.2 LTS"
     NAME="Ubuntu"
     VERSION="18.04.2 LTS (Bionic Beaver)"
     ID=ubuntu
     ID_LIKE=debian
     PRETTY_NAME="Ubuntu 18.04.2 LTS"
     VERSION_ID="18.04"
     HOME_URL="https://www.ubuntu.com/"
     SUPPORT_URL="https://help.ubuntu.com/"
     BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
     PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
     VERSION_CODENAME=bionic
     UBUNTU_CODENAME=bionic
     Linux dgx1-zqureshi 4.15.0-55-generic #60-Ubuntu SMP Tue Jul 2 18:22:20 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
     
     ***GPU Information***
     Mon Aug 19 13:11:14 2019
     +-----------------------------------------------------------------------------+
     | NVIDIA-SMI 410.48                 Driver Version: 410.48                    |
     |-------------------------------+----------------------+----------------------+
     | GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
     | Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
     |===============================+======================+======================|
     |   0  Tesla V100-SXM2...  On   | 00000000:05:00.0 Off |                    0 |
     | N/A   33C    P0    44W / 160W |      0MiB / 16130MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     |   1  Tesla V100-SXM2...  On   | 00000000:06:00.0 Off |                    0 |
     | N/A   34C    P0    42W / 160W |      0MiB / 16130MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     |   2  Tesla V100-SXM2...  On   | 00000000:09:00.0 Off |                    0 |
     | N/A   34C    P0    45W / 160W |      0MiB / 16130MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     |   3  Tesla V100-SXM2...  On   | 00000000:0A:00.0 Off |                    0 |
     | N/A   30C    P0    44W / 160W |      0MiB / 16130MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     |   4  Tesla V100-SXM2...  On   | 00000000:85:00.0 Off |                    0 |
     | N/A   32C    P0    43W / 160W |      0MiB / 16130MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     |   5  Tesla V100-SXM2...  On   | 00000000:86:00.0 Off |                    0 |
     | N/A   34C    P0    41W / 160W |      0MiB / 16130MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     |   6  Tesla V100-SXM2...  On   | 00000000:89:00.0 Off |                    0 |
     | N/A   39C    P0    44W / 160W |      0MiB / 16130MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     |   7  Tesla V100-SXM2...  On   | 00000000:8A:00.0 Off |                    0 |
     | N/A   32C    P0    43W / 160W |      0MiB / 16130MiB |      0%      Default |
     +-------------------------------+----------------------+----------------------+
     
     +-----------------------------------------------------------------------------+
     | Processes:                                                       GPU Memory |
     |  GPU       PID   Type   Process name                             Usage      |
     |=============================================================================|
     |  No running processes found                                                 |
     +-----------------------------------------------------------------------------+
     
     ***CPU***
     Architecture:        x86_64
     CPU op-mode(s):      32-bit, 64-bit
     Byte Order:          Little Endian
     CPU(s):              80
     On-line CPU(s) list: 0-79
     Thread(s) per core:  2
     Core(s) per socket:  20
     Socket(s):           2
     NUMA node(s):        2
     Vendor ID:           GenuineIntel
     CPU family:          6
     Model:               79
     Model name:          Intel(R) Xeon(R) CPU E5-2698 v4 @ 2.20GHz
     Stepping:            1
     CPU MHz:             1197.510
     CPU max MHz:         3600.0000
     CPU min MHz:         1200.0000
     BogoMIPS:            4390.09
     Virtualization:      VT-x
     L1d cache:           32K
     L1i cache:           32K
     L2 cache:            256K
     L3 cache:            51200K
     NUMA node0 CPU(s):   0-19,40-59
     NUMA node1 CPU(s):   20-39,60-79
     Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb cat_l3 cdp_l3 invpcid_single pti intel_ppin ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid fsgsbase tsc_adjust bmi1 hle avx2 smep bmi2 erms invpcid rtm cqm rdt_a rdseed adx smap intel_pt xsaveopt cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local dtherm ida arat pln pts md_clear flush_l1d
     
     ***CMake***
     /home/zqureshi/anaconda2/envs/cudf_dev/bin/cmake
     cmake version 3.15.1
     
     CMake suite maintained and supported by Kitware (kitware.com/cmake).
     
     ***g++***
     /usr/bin/g++
     g++ (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
     Copyright (C) 2017 Free Software Foundation, Inc.
     This is free software; see the source for copying conditions.  There is NO
     warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
     
     
     ***nvcc***
     /usr/local/cuda/bin/nvcc
     nvcc: NVIDIA (R) Cuda compiler driver
     Copyright (c) 2005-2018 NVIDIA Corporation
     Built on Sat_Aug_25_21:08:01_CDT_2018
     Cuda compilation tools, release 10.0, V10.0.130
     
     ***Python***
     /home/zqureshi/anaconda2/envs/cudf_dev/bin/python
     Python 3.7.3
     
     ***Environment Variables***
     PATH                            : /home/zqureshi/anaconda2/envs/cudf_dev/bin:/home/zqureshi/anaconda2/condabin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
     LD_LIBRARY_PATH                 : /usr/local/cuda/lib64:
     NUMBAPRO_NVVM                   :
     NUMBAPRO_LIBDEVICE              :
     CONDA_PREFIX                    : /home/zqureshi/anaconda2/envs/cudf_dev
     PYTHON_PATH                     :
     
     ***conda packages***
     /home/zqureshi/anaconda2/condabin/conda
     # packages in environment at /home/zqureshi/anaconda2/envs/cudf_dev:
     #
     # Name                    Version                   Build  Channel
     _libgcc_mutex             0.1                        main
     alabaster                 0.7.12                     py_0    conda-forge
     appdirs                   1.4.3                      py_1    conda-forge
     arrow-cpp                 0.14.1           py37h6b969ab_0    conda-forge
     asn1crypto                0.24.0                py37_1003    conda-forge
     aspy.yaml                 1.3.0                      py_0    conda-forge
     atomicwrites              1.3.0                      py_0    conda-forge
     attrs                     19.1.0                     py_0    conda-forge
     aws-sam-translator        1.13.1                     py_0    conda-forge
     aws-xray-sdk              0.95                       py_0    conda-forge
     babel                     2.7.0                      py_0    conda-forge
     backcall                  0.1.0                      py_0    conda-forge
     backports                 1.0                        py_2    conda-forge
     backports.tempfile        1.0                        py_0    conda-forge
     backports.weakref         1.0.post1             py37_1000    conda-forge
     black                     19.3b0                     py_0    conda-forge
     bleach                    3.1.0                      py_0    conda-forge
     bokeh                     1.3.4                    py37_0    conda-forge
     boost-cpp                 1.70.0               h8e57a91_2    conda-forge
     boto                      2.49.0                     py_0    conda-forge
     boto3                     1.9.208                    py_0    conda-forge
     botocore                  1.12.208                   py_0    conda-forge
     brotli                    1.0.7             he1b5a44_1000    conda-forge
     bzip2                     1.0.8                h516909a_0    conda-forge
     c-ares                    1.15.0            h516909a_1001    conda-forge
     ca-certificates           2019.6.16            hecc5488_0    conda-forge
     cached-property           1.5.1                      py_0    conda-forge
     certifi                   2019.6.16                py37_1    conda-forge
     cffi                      1.12.3           py37h8022711_0    conda-forge
     cfgv                      2.0.1                      py_0    conda-forge
     cfn-lint                  0.23.2                     py_0    conda-forge
     chardet                   3.0.4                 py37_1003    conda-forge
     click                     7.0                        py_0    conda-forge
     cloudpickle               1.2.1                      py_0    conda-forge
     cmake                     3.15.1               hf94ab9c_0    conda-forge
     commonmark                0.9.0                      py_0    conda-forge
     cookies                   2.2.1                      py_0    conda-forge
     cryptography              2.7              py37h72c5cf5_0    conda-forge
     cudatoolkit               10.0.130                      0
     cudf                      0.10.0a0+611.g66aeacf4d          pypi_0    pypi
     curl                      7.65.3               hf8cf82a_0    conda-forge
     cython                    0.29.13          py37he1b5a44_0    conda-forge
     cytoolz                   0.10.0           py37h516909a_0    conda-forge
     dask                      2.2.0+44.g46525a36          pypi_0    pypi
     decorator                 4.4.0                      py_0    conda-forge
     defusedxml                0.5.0                      py_1    conda-forge
     distributed               2.2.0+25.g63021750          pypi_0    pypi
     dlpack                    0.2                  he1b5a44_0    conda-forge
     docker-py                 4.0.2                    py37_0    conda-forge
     docker-pycreds            0.4.0                      py_0    conda-forge
     docutils                  0.14                  py37_1001    conda-forge
     double-conversion         3.1.5                he1b5a44_1    conda-forge
     ecdsa                     0.13                       py_0    conda-forge
     editdistance              0.5.3            py37hf484d3e_0    conda-forge
     entrypoints               0.3                   py37_1000    conda-forge
     expat                     2.2.5             he1b5a44_1003    conda-forge
     fastavro                  0.22.3           py37h516909a_0    conda-forge
     flake8                    3.7.8                    py37_1    conda-forge
     flask                     1.1.1                      py_1    conda-forge
     flatbuffers               1.11.0               he1b5a44_0    conda-forge
     freetype                  2.10.0               he983fc9_1    conda-forge
     fsspec                    0.4.1                      py_0    conda-forge
     future                    0.17.1                py37_1000    conda-forge
     gflags                    2.2.2             he1b5a44_1001    conda-forge
     glog                      0.4.0                he1b5a44_1    conda-forge
     gmp                       6.1.2             hf484d3e_1000    conda-forge
     grpc-cpp                  1.22.0               h18db393_0    conda-forge
     heapdict                  1.0.0                 py37_1000    conda-forge
     httpretty                 0.9.6                      py_0    conda-forge
     hypothesis                4.32.3                   py37_0    conda-forge
     icu                       64.2                 he1b5a44_0    conda-forge
     identify                  1.4.6                      py_0    conda-forge
     idna                      2.8                   py37_1000    conda-forge
     imagesize                 1.1.0                      py_0    conda-forge
     importlib_metadata        0.18                     py37_0    conda-forge
     ipykernel                 5.1.2            py37h5ca1d4c_0    conda-forge
     ipython                   7.7.0            py37h5ca1d4c_0    conda-forge
     ipython_genutils          0.2.0                      py_1    conda-forge
     isort                     4.3.21                   py37_0    conda-forge
     itsdangerous              1.1.0                      py_0    conda-forge
     jedi                      0.15.1                   py37_0    conda-forge
     jinja2                    2.10.1                     py_0    conda-forge
     jmespath                  0.9.4                      py_0    conda-forge
     jpeg                      9c                h14c3975_1001    conda-forge
     jsondiff                  1.1.2                      py_0    conda-forge
     jsonpatch                 1.24                       py_0    conda-forge
     jsonpickle                1.2                        py_0    conda-forge
     jsonpointer               2.0                        py_0    conda-forge
     jsonschema                3.0.2                    py37_0    conda-forge
     jupyter_client            5.3.1                      py_0    conda-forge
     jupyter_core              4.4.0                      py_0    conda-forge
     krb5                      1.16.3            h05b26f9_1001    conda-forge
     libblas                   3.8.0               12_openblas    conda-forge
     libcblas                  3.8.0               12_openblas    conda-forge
     libcurl                   7.65.3               hda55be3_0    conda-forge
     libedit                   3.1.20170329      hf8c457e_1001    conda-forge
     libevent                  2.1.10               h72c5cf5_0    conda-forge
     libffi                    3.2.1             he1b5a44_1006    conda-forge
     libgcc-ng                 9.1.0                hdf63c60_0
     libgfortran-ng            7.3.0                hdf63c60_0
     liblapack                 3.8.0               12_openblas    conda-forge
     libnvstrings              0.9.0a1             cuda10.0_64    rapidsai-nightly
     libopenblas               0.3.7                h6e990d7_0    conda-forge
     libpng                    1.6.37               hed695b0_0    conda-forge
     libprotobuf               3.8.0                h8b12597_0    conda-forge
     librmm                    0.9.0a1             cuda10.0_14    rapidsai-nightly
     libsodium                 1.0.17               h516909a_0    conda-forge
     libssh2                   1.8.2                h22169c7_2    conda-forge
     libstdcxx-ng              9.1.0                hdf63c60_0
     libtiff                   4.0.10            h57b8799_1003    conda-forge
     libuv                     1.30.1               h516909a_0    conda-forge
     llvmlite                  0.29.0           py37hfd453ef_1    conda-forge
     locket                    0.2.0                      py_2    conda-forge
     lz4-c                     1.8.3             he1b5a44_1001    conda-forge
     markdown                  2.6.11                   pypi_0    pypi
     markupsafe                1.1.1            py37h14c3975_0    conda-forge
     mccabe                    0.6.1                      py_1    conda-forge
     mistune                   0.8.4           py37h14c3975_1000    conda-forge
     mock                      3.0.5                    py37_0    conda-forge
     more-itertools            7.2.0                      py_0    conda-forge
     moto                      1.3.8                      py_1    conda-forge
     msgpack-python            0.6.1            py37h6bb024c_0    conda-forge
     nbconvert                 5.6.0                      py_0    conda-forge
     nbformat                  4.4.0                      py_1    conda-forge
     nbsphinx                  0.4.2                      py_0    conda-forge
     ncurses                   6.1               hf484d3e_1002    conda-forge
     nodeenv                   1.3.3                      py_0    conda-forge
     notebook                  6.0.0                    py37_0    conda-forge
     numba                     0.45.1           py37hb3f55d8_0    conda-forge
     numpy                     1.17.0           py37h95a1406_0    conda-forge
     numpydoc                  0.9.1                      py_0    conda-forge
     nvstrings                 0.9.0a1                 py37_64    rapidsai-nightly
     olefile                   0.46                       py_0    conda-forge
     openssl                   1.1.1c               h516909a_0    conda-forge
     packaging                 19.0                       py_0    conda-forge
     pandas                    0.24.2           py37hb3f55d8_0    conda-forge
     pandoc                    1.19.2                        0    conda-forge
     pandocfilters             1.4.2                      py_1    conda-forge
     parquet-cpp               1.5.1                         2    conda-forge
     parso                     0.5.1                      py_0    conda-forge
     partd                     1.0.0                      py_0    conda-forge
     pexpect                   4.7.0                    py37_0    conda-forge
     pickleshare               0.7.5                 py37_1000    conda-forge
     pillow                    6.1.0            py37h6b7be26_1    conda-forge
     pip                       19.2.2                   py37_0    conda-forge
     pluggy                    0.12.0                     py_0    conda-forge
     pre_commit                1.18.1                   py37_0    conda-forge
     prometheus_client         0.7.1                      py_0    conda-forge
     prompt_toolkit            2.0.9                      py_0    conda-forge
     psutil                    5.6.3            py37h516909a_0    conda-forge
     ptyprocess                0.6.0                   py_1001    conda-forge
     py                        1.8.0                      py_0    conda-forge
     pyarrow                   0.14.1           py37h8b68381_0    conda-forge
     pycodestyle               2.5.0                      py_0    conda-forge
     pycparser                 2.19                     py37_1    conda-forge
     pycryptodome              3.8.2            py37he80fd80_0    conda-forge
     pyflakes                  2.1.1                      py_0    conda-forge
     pygments                  2.4.2                      py_0    conda-forge
     pyopenssl                 19.0.0                   py37_0    conda-forge
     pyparsing                 2.4.2                      py_0    conda-forge
     pyrsistent                0.15.4           py37h516909a_0    conda-forge
     pysocks                   1.7.0                    py37_0    conda-forge
     pytest                    5.0.1                    py37_1    conda-forge
     python                    3.7.3                h33d41f4_1    conda-forge
     python-dateutil           2.8.0                      py_0    conda-forge
     python-jose               2.0.2                      py_0    conda-forge
     pytz                      2019.2                     py_0    conda-forge
     pyyaml                    5.1.2            py37h516909a_0    conda-forge
     pyzmq                     18.0.2           py37h1768529_2    conda-forge
     rapidjson                 1.1.0             he1b5a44_1002    conda-forge
     re2                       2019.08.01           he1b5a44_0    conda-forge
     readline                  8.0                  hf8c457e_0    conda-forge
     recommonmark              0.6.0                      py_0    conda-forge
     requests                  2.22.0                   py37_1    conda-forge
     responses                 0.9.0                      py_0    conda-forge
     rhash                     1.3.6             h14c3975_1001    conda-forge
     rmm                       0.9.0a1                 py37_14    rapidsai-nightly
     s3fs                      0.3.3                      py_0    conda-forge
     s3transfer                0.2.1                    py37_0    conda-forge
     send2trash                1.5.0                      py_0    conda-forge
     setuptools                41.0.1                   py37_0    conda-forge
     six                       1.12.0                py37_1000    conda-forge
     snappy                    1.1.7             he1b5a44_1002    conda-forge
     snowballstemmer           1.9.0                      py_0    conda-forge
     sortedcontainers          2.1.0                      py_0    conda-forge
     sphinx                    2.1.2                      py_0    conda-forge
     sphinx-markdown-tables    0.0.9                    pypi_0    pypi
     sphinx_rtd_theme          0.4.3                      py_0    conda-forge
     sphinxcontrib-applehelp   1.0.1                      py_0    conda-forge
     sphinxcontrib-devhelp     1.0.1                      py_0    conda-forge
     sphinxcontrib-htmlhelp    1.0.2                      py_0    conda-forge
     sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
     sphinxcontrib-qthelp      1.0.2                      py_0    conda-forge
     sphinxcontrib-serializinghtml 1.1.1                      py_0    conda-forge
     sphinxcontrib-websupport  1.1.2                      py_0    conda-forge
     sqlite                    3.29.0               hcee41ef_0    conda-forge
     tblib                     1.4.0                      py_0    conda-forge
     terminado                 0.8.2                    py37_0    conda-forge
     testpath                  0.4.2                   py_1001    conda-forge
     thrift-cpp                0.12.0            hf3afdfd_1004    conda-forge
     tk                        8.6.9             hed695b0_1002    conda-forge
     toml                      0.10.0                     py_0    conda-forge
     toolz                     0.10.0                     py_0    conda-forge
     tornado                   6.0.3            py37h516909a_0    conda-forge
     traitlets                 4.3.2                 py37_1000    conda-forge
     uriparser                 0.9.3                he1b5a44_1    conda-forge
     urllib3                   1.25.3                   py37_0    conda-forge
     virtualenv                16.7.2                   py37_0    conda-forge
     wcwidth                   0.1.7                      py_1    conda-forge
     webencodings              0.5.1                      py_1    conda-forge
     websocket-client          0.56.0                   py37_0    conda-forge
     werkzeug                  0.15.5                     py_0    conda-forge
     wheel                     0.33.4                   py37_0    conda-forge
     wrapt                     1.11.2           py37h516909a_0    conda-forge
     xmltodict                 0.12.0                     py_0    conda-forge
     xz                        5.2.4             h14c3975_1001    conda-forge
     yaml                      0.1.7             h14c3975_1001    conda-forge
     zeromq                    4.3.2                he1b5a44_2    conda-forge
     zict                      1.0.0                      py_0    conda-forge
     zipp                      0.5.2                      py_0    conda-forge
     zlib                      1.2.11            h516909a_1005    conda-forge
     zstd                      1.4.0                h3b9ef0a_0    conda-forge
     
</pre></details>

**Additional context**

**Comment by kkraus14** [comment]
@ZaidQureshi is there any data or reproducer that you can share? If not could you share which column(s) have incorrect data and what their data type / encoding / compression in the ORC file are?

**Comment by ZaidQureshi** [comment]
I am using the NYC Yellow and Green Taxi ride dataset. I converted it using the steps provided [here](https://tech.marksblogg.com/faster-csv-to-orc-conversions.html) using spark-sql, except I don't use any compression. The data types I have tried are of type DOUBLE and SMALLINT. The SMALLINT columns are all encoded using ORC's rlev2. Like I said all the tested columns have values for all rows. 2 of the columns that I am interested in from this data are trip_distance and pickup_nyct2010_gid.
